template/layout.html.twig

```diff
@@ -2,22 +2,59 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
     <title>Mood - Colla.io</title>
-    <link rel="stylesheet" href="https://unpkg.com/purecss@1.0.0/build/pure-min.css"
-          integrity="sha384-nn4HPE8lTHyVtfCBi5yW9d20FjT8BJwUXyWZT9InLYax14RDjBj46LmSztkmNP9w" crossorigin="anonymous">
-    <script src="https://cdn.jsdelivr.net/npm/jquery@3.4.1/dist/jquery.min.js"
-            integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo=" crossorigin="anonymous"></script>
+
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+          integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous">
+
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/flatpickr/4.5.7/flatpickr.min.css">
 </head>
 <body>
-<div class="pure-g">
-    <div class="pure-u-1-8"></div>
-    <div class="pure-u-3-8">
-        <div style="box-shadow: -1px 1px 17px #ccc; padding: 10px; margin-top:10px;">
-            {% block content %}{% endblock %}
-        </div>
-    </div>
-    <div class="pure-u-4-8"></div>
-</div>
+
+<header>
+    <nav class="navbar navbar-expand-md navbar-dark bg-dark">
+        <a class="navbar-brand" href="{{ path_for('index') }}">Team Mood</a>
+    </nav>
+</header>
+
+<main role="main" class="container pt-3">
+    {% block content %}{% endblock %}
+</main>
+
+<script>
+    // Disabling form submissions if there are invalid fields.
+    (function () {
+        'use strict';
+        window.addEventListener('load', function () {
+            // Fetch all the forms we want to apply custom Bootstrap validation styles to.
+            var forms = document.getElementsByClassName('needs-validation');
+
+            // Loop over them and prevent submission.
+            var validation = Array.prototype.filter.call(forms, function (form) {
+                form.addEventListener('submit', function (event) {
+                    if (form.checkValidity() === false) {
+                        event.preventDefault();
+                        event.stopPropagation();
+                    }
+                    form.classList.add('was-validated');
+                }, false);
+            });
+        }, false);
+    })();
+</script>
+
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js"
```

Comment:
I think it's not required any more after you've refactored the form logic to vanilla js?

Reply:
I would be more than happy to get rid of it, but unfortunately bootstrap still requires at least the slim version.